dbld/images/helpers/dependencies.sh

```diff
@@ -107,14 +107,17 @@ function install_yum_packages {
 
 function install_pip_packages {
     case "${OS_PLATFORM}" in
+        centos-6)
+            pip install --upgrade pip==9.0.3
+            ;;
         centos-7)
             python -m pip install --upgrade pip==9.0.3
             ;;
         *)
             python -m pip install --upgrade pip
             ;;
     esac
-    filter_packages_by_platform /helpers/pip_packages.manifest | xargs python -m pip install -U
+    filter_packages_by_platform /helpers/pip_packages.manifest | xargs pip install -U
```

Comment:
Could you explain this change? Why do we need to use `pip install` instead of `python -m pip install`?